deepfence_server/pkg/registry/acr/client.go

```diff
@@ -11,39 +12,103 @@ import (
 	"github.com/deepfence/ThreatMapper/deepfence_utils/log"
 )
 
-var client = &http.Client{Timeout: 10 * time.Second}
+const (
+	PerPageCount       = 100
+	ParallelImageFetch = 10
+)
+
+var (
+	client                = &http.Client{Timeout: 10 * time.Second}
+	parllelImageProcessor *tunny.Pool
+	queue                 chan []model.IngestedContainerImage
```

Comment:
We need to make this a buffered channel. 
We can give this a buffer size of 100